aicsimageio/tests/readers/test_glob_reader.py

```diff
@@ -21,10 +21,7 @@ def check_values(
 ) -> None:
     for i, s in enumerate(reader.scenes):
         reader.set_scene(s)
-        assert np.all(
-            reference.isel(S=i).data == reader.xarray_dask_data.data
-        ).compute()
-        assert np.all(reference.isel(S=i).data == reader.xarray_data.data)
+        np.testing.assert_array_equal(reference.isel(S=i).data, reader.xarray_data.data)
```

Comment:
why remove the dask test too?

Reply:
np.testing.assert_array_equal(reference.isel(S=i).data, reader.xarray_data.data) does what the previous lines were doing